### DISCUSSION TITLE
Table 2, Panel 1

### DISCUSSION THREAD

**Discussion by hasanmetin** [cssi-pi-community/2022-meeting#50] [Panel 1 roundtable (Monday morning)]
Our discussion mainly focused on maintenance and sustainability of academic/scientific software.  Following are the main points discussed:

- Open source software has been pushed by NSF, DOE and other agencies early on, but they are not keen on funding maintenance projects.

- For sustainability, some software packages charge a license fee which is typically a nominal fee for academics. However, this is not typically welcome in the academic community, a good example is AMBER (charges fees) vs LAMMPS or GROMACS (which are free). We typically prefer free software and do not budget license fees in our projects, but maybe we should in the interest of supporting sustainability. Donation can also be a model for open source software sustainability, but donations cannot be included as a budget line in federal funding projects. Also, things get complicated at the institutional level in terms of the acceptance and distribution of funds (donation or license).

- Citation is an important metric in demonstrating the impact of open source software. In large software projects, individual packages may get overlooked - a citation generator can allow developers to get the credit they deserve.

- For research professionals (postdocs or research software engineers), there was an idea to reduce overheads so that better salary can be given to them. However, this might be hard to accomplish because overhead amount is typically an institutional decision to cover facilities and administration costs and federal agencies do not have much say in this.

We also discussed diversity issues:
- While there has been a fair share of discussion on the representation of women in CI, we noted that we have not seen any people of color in the meeting which we believe is something to ponder upon as well. Are the issues of recruitment and retention of black people similar to those of women or are there other things to consider?

- Recruitment pool is actually a problem for many CI disciplines. Even when it is not, female students might actually prefer more hands-on groups (experimental in life sciences or mechanical engineering for instance, or vision/pattern recognition in computer science) over CI areas. This could be because those groups are typically larger (hence there is a better support group) or they do not require the additional technical training that they have to go through in CI related fields.

- One group member said that they have had very good success in announcing well-defined tasks within their project to women in computing groups, to clubs by underrepresented minorities, and as term projects. They also had some luck in retaining some of those students longer term.

- To aid these efforts, REU supplements can also be used. Tevfik Kosar, the PM at our table, added that "internship supplements" are available too, which might be another way to support/retain undergrad students. Internship supplement was something not well known among our participants.
-

**Comment by tevfikkosar** [comment]
A minor correction to the last bullet in this summary: NSF INTERN Supplements are available only for graduate students. Here is the link for those who are interested in learning more: https://www.nsf.gov/pubs/2021/nsf21013/nsf21013.jsp
